Mathlib/NumberTheory/LegendreSymbol/AddCharacter.lean

```diff
@@ -332,4 +333,10 @@ lemma FiniteField.primitiveChar_to_Complex_isPrimitive :
 
 end Field
 
+/-- The standard additive character `ZMod N → ℂ` is primitive. -/
+lemma isPrimitive_stdAddChar (N : ℕ) [NeZero N] :
+    IsPrimitive (ZMod.stdAddChar (N := N)) := by
+  refine zmod_char_primitive_of_eq_one_only_at_zero _ _ (fun t ht ↦ ?_)
+  rwa [← (ZMod.stdAddChar (N := N)).map_zero_eq_one, ZMod.injective_stdAddChar.eq_iff] at ht
+
```

Comment:
I think it would make sense to move this, together with the section `Field` above it, to a separate file, in order to reduce the imports for `NumberTheory.LegendreSymbol.AddCharacter` (`import Mathlib.Analysis.Complex.Polynomial` is only needed for these results).

Reply:
OK; let's keep this in mind for later.